crawl-ref/source/player.cc

```diff
@@ -6009,36 +6009,13 @@ int player::armour_class_with_specific_items(vector<const item_def *> items) con
   * useful when the AC roll is inferior to it. Therefore a higher GDR means
   * more damage reduced, but also more often.
   *
-  * \f[ GDR = 14 \times (base\_AC - 2)^\frac{1}{2} \f]
+  * \f[ GDR = 18 \times (AC)^\frac{1}{2} \f]
```

Comment:
Should it be ``frac{1}{4}`` instead of ``frac{1}{2}``?

Reply:
Yep, you're quite right, thanks!